lib/components/note-preview/index.tsx

```diff
@@ -134,7 +137,7 @@ export const NotePreview: FunctionComponent<Props> = ({
     previewNode.current?.addEventListener('click', handleClick, true);
     return () =>
       previewNode.current?.removeEventListener('click', handleClick, true);
-  }, []);
+  }, [note]);
```

Comment:
good catch! though while we're at it it looks more like the dependencies are `note.content` and `searchQuery` so maybe we should follow-up and set those two specifically. thoughts? obviously if the `note` changes the `note.content` might be different but only indirectly so.

Reply:
searchQuery is not used in that useEffect. I do not think it is needed as a dependency. Changing the dependency to a note.content is a good change.

Reply:
https://github.com/Automattic/simplenote-electron/pull/2461